corehq/apps/app_manager/urls.py

```diff
@@ -171,6 +171,7 @@
     url(r'^schema/(?P<app_id>[\w-]+)/form/(?P<form_unique_id>[\w-]+)/$',
         get_form_data_schema, name='get_form_data_schema'),
     url(r'^new_module/(?P<app_id>[\w-]+)/$', new_module, name='new_module'),
+    url(r'^new_module/(?P<app_id>[\w-]+)/(?P<case_type>[\w-]+)/$', new_module, name='new_module_with_case_type'),
```

Comment:
It doesn't look like this is called.

Reply:
It is called conditionally, when user click on "Create Case List", I add [this hidden field](https://github.com/dimagi/commcare-hq/blob/da2a0b721072fbca43b10e053b71fb6e72e63646/corehq/apps/app_manager/static/app_manager/js/app_manager.js#L626) and set the value, then the case type value will be submitted along with [this form](https://github.com/dimagi/commcare-hq/blob/5b1b2c1bf98bd5dbd492604e7a6f9bcb676bddea/corehq/apps/app_manager/templates/app_manager/partials/menu/appnav_menu.html#L45-L53)

Reply:
hmm, I see how you're adding the hidden input, but I don't see the action of the form changing, I think it's still submitting to the existing `new_module` URL. (And that's fine: since you're adding the case type as a POST variable, there's no need to also put it in the URL.) Or am I missing something else?

Reply:
Ahhhhhhh you're right. Thanks for pointing it out!! 6c0732a79baace88bcfb4b44aaadb1e8a5b97c8d